packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -535,25 +546,8 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends React.Componen
       this.setState({ isFocused: true });
       this.selection.setAllSelected(false);
 
-      if (
-        this.input.current &&
-        this.input.current.value === '' &&
-        // tslint:disable-next-line:deprecation
-        (this.props.onEmptyResolveSuggestions || this.props.onEmptyInputFocus) &&
-        !this._requestSuggestionsOnClick
-      ) {
-        this.onEmptyInputFocus();
-        this.setState({
-          isMostRecentlyUsedVisible: true,
-          moreSuggestionsAvailable: false,
-          suggestionsVisible: true,
-        });
-      } else if (this.input.current && this.input.current.value && !this._requestSuggestionsOnClick) {
-        this.setState({
-          isMostRecentlyUsedVisible: false,
-          suggestionsVisible: true,
-        });
-      }
+      this._userTriggeredSuggestions();
+
       if (this.props.inputProps && this.props.inputProps.onFocus) {
         this.props.inputProps.onFocus(ev as React.FocusEvent<HTMLInputElement>);
       }
```

Comment:
```suggestion
      this.props.inputProps?.onFocus?(ev as React.FocusEvent<HTMLInputElement>);
```

Reply:
Did we get these fixed? I think it was an issue with babel if I remember correctly.